pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -116,6 +127,198 @@ func TestShowChangefeedJobsBasic(t *testing.T) {
 	cdcTest(t, testFn, feedTestOmitSinks("webhook", "sinkless"), feedTestNoExternalConnection)
 }
 
+func TestShowChangefeedJobsShowsHighWaterAfterCheckpoint(t *testing.T) {
```

Comment:
This whole test is mostly copied verbatim from here (test called "TestChangefeedBackfillCheckpoint"): https://github.com/cockroachdb/cockroach/blob/7144abb5b7e875d3785aeac291ccc60d945489e0/pkg/ccl/changefeedccl/changefeed_test.go#L7105
Just with an additional "SELECT high_water_timestamp, readable_high_water_timestamp from [SHOW CHANGEFEED JOBS]" after the triggered checkpoint.
Without that, these columns come back NULL and I get an error when trying to scan. 
This seems like it should be able to be pared down, but removing even the "drainUntilTimestamp" calls after the new select seems to fail the test.